dbt/scripts/run_iasworld_data_tests.py

```diff
@@ -693,33 +700,13 @@ def main() -> None:
         formatter_class=argparse.RawTextHelpFormatter,
     )
 
-    parser.add_argument(
-        "--run-results",
-        required=False,
-        default=os.path.join("target", "run_results.json"),
-        help=(
-            "The local path to a run_results.json file generated by a "
-            "`dbt test --store-failures` run. Defaults to "
-            "'./target/run_results.json'."
-        ),
-    )
-    parser.add_argument(
-        "--manifest",
-        required=False,
-        default=os.path.join("target", "manifest.json"),
-        help=(
-            "The local path to a manifest.json file with the compiled dbt "
-            "DAG, generated by all dbt CLI commands (including `dbt test`). "
-            "Defaults to './target/manifest.json'."
-        ),
-    )
```

Comment:
Since the script is now responsible for running tests, we no longer need to expose config params to point to the location of the run results and manifest files.